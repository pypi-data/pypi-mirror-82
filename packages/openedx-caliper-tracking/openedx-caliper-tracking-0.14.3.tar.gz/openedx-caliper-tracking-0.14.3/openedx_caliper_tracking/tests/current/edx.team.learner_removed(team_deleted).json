{
  "accept_language": "en-US,en;q=0.9",
  "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36",
  "context": {
    "course_id": "course-v1:edX+DemoX+Demo_Course",
    "org_id": "edX",
    "path": "/api/team/v0/team_membership/check-3429fc5983a84c8c8366a4477b03d91c,honor",
    "user_id": 2
  },
  "event": {
    "remove_method": "team_deleted",
    "team_id": "check-3429fc5983a84c8c8366a4477b03d91c",
    "user_id": 6
  },
  "event_source": "server",
  "event_type": "edx.team.learner_removed",
  "host": "e5d2c598af1a",
  "ip": "172.18.0.1",
  "name": "edx.team.learner_removed",
  "page": null,
  "referer": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/",
  "session": "dff5c86a2aa1ebe2ea76bfd226e098a3",
  "time": "2019-01-01T07:05:11.198463+00:00",
  "username": "verified"
}
