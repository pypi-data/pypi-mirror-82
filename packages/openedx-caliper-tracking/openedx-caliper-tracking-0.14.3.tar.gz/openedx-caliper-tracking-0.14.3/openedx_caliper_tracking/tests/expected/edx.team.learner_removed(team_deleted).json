{
  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
  "action": "Removed",
  "actor": {
    "id": "http://localhost:18000/u/verified",
    "name": "verified",
    "type": "Person"
  },
  "eventTime": "2019-01-01T07:05:11.198Z",
  "extensions": {
    "extra_fields": {
      "accept_language": "en-US,en;q=0.9",
      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36",
      "course_id": "course-v1:edX+DemoX+Demo_Course",
      "event_source": "server",
      "event_type": "edx.team.learner_removed",
      "host": "e5d2c598af1a",
      "ip": "172.18.0.1",
      "org_id": "edX",
      "page": null,
      "path": "/api/team/v0/team_membership/check-3429fc5983a84c8c8366a4477b03d91c,honor",
      "session": "dff5c86a2aa1ebe2ea76bfd226e098a3",
      "user_id": 2
    }
  },
  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
  "object": {
    "extensions": {
      "remove_method": "team_deleted"
    },
    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/",
    "member": {
      "extensions": {
        "user_id": 6
      },
      "id": "http://localhost:18000/u/honor",
      "name": "honor",
      "type": "Person"
    },
    "organization": {
      "extensions": {
        "team_id": "check-3429fc5983a84c8c8366a4477b03d91c"
      },
      "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/",
      "type": "Group"
    },
    "type": "Membership"
  },
  "referrer": {
    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/",
    "type": "WebPage"
  },
  "type": "Event"
}
